doc/guide/guide-overview.rst

```diff
@@ -4,25 +4,7 @@
 Guide Overview
 ******************
 
-The goal of this guide is to introduce you to the basic structures and functions that make up QuTiP. This guide is divided up into several 
-sections, each highlighting a specific set of functionalities. In combination with the examples that can be found on the project 
-web page `https://qutip.org/tutorials.html <https://qutip.org/tutorials.html>`_, this guide should provide a more or less complete overview 
+The goal of this guide is to introduce you to the basic structures and functions that make up QuTiP. This guide is divided up into several
+sections, each highlighting a specific set of functionalities. In combination with the examples that can be found on the project
+web page `https://qutip.org/tutorials.html <https://qutip.org/tutorials.html>`_, this guide should provide a more or less complete overview
 of QuTip. We also provide the API documentation in :ref:`apidoc`.
-
```

Comment:
It would be nice to keep the overview and there is code for generating it in `doc/QuTiP_tree_plot`. Perhaps that could be updated and perhaps automated a bit more?

Reply:
This circular format might have been nice 10 years ago, but with the size that qutip is now, it's an unreadable mess... Just updating it is not enough to make it worth keeping.

Reply:
Then perhaps remove the rest of it? I.e. `doc/QuTiP_tree_plot/`.

Reply:
Thanks!